app/models/miq_server/ntp_management.rb

```diff
@@ -47,7 +46,18 @@ def ntp_reload(ntp_settings = server_ntp_settings)
       return
     end
     _log.info("Synchronizing ntp settings: #{ntp_settings.inspect}")
-    MiqNtp.sync_settings(ntp_settings)
+    apply_ntp_server_settings(ntp_settings)
     @ntp_settings = ntp_settings
   end
+
+  def apply_ntp_server_settings(settings)
+    return unless Sys::Platform::IMPL == :linux
```

Comment:
Not sure, I just tried it keep it as close to what was being done [here](https://github.com/ManageIQ/manageiq/pull/5705/files#diff-a26224b1e4d800d3637c07c373128f5cL7).
This will probably only be an issue when running in development (not on an appliance) right?